### ISSUE TITLE
Traffic lights not visible

### ISSUE THREAD

**Issue by dpo** [zeit/hyper#848]
I'm using Hyper 0.8.1.737 on OSX 10.9. The Hyper window has no traffic lights:

![screen shot 2016-10-10 at 20 03 33](https://cloud.githubusercontent.com/assets/38760/19254718/c56496d0-8f24-11e6-9b0a-8cf3f3f7bbdf.png)

It looks as if the tab is covering them because clicking at the place where they should be either minimizes, maximizes or closes the window. I did install `hyperclean` but disabled it along with all other plugins. The traffic lights were never visible (even before I installed `hyperclean`).

As a side note, when doing a full reload, the window reappears completely transparent, but with traffic lights (the ducks are on the wallpaper):

![screen shot 2016-10-10 at 20 06 43](https://cloud.githubusercontent.com/assets/38760/19254743/1f9c0674-8f25-11e6-8b6c-963c6ef80b9d.png)

I tried deleting my `~/.hyper.js`, to no avail.

Thanks.

**Comment by OJFord** [comment]
I don't have this issue, same Hyper version, but macOS 10.12 not 10.9. Have they ever been there?

**Comment by dpo** [comment]
No, they were never there (except when I do a full reload and the window is transparent). I might be something with OSX 10.9.

**Comment by OJFord** [comment]
> might be something with OSX 10.9

Sounds like it - maybe try fiddling with the CSS in developer tools pane. Do you have any other Electron apps (e.g. Atom, Slack) that display them okay?

**Comment by dpo** [comment]
Yes Atom is ok. I'm not very familiar with those things. Could you provide specific suggestions?

**Comment by OJFord** [comment]
Not really, I just thought that's what I would fiddle with (and I really would just be fiddling) if I could see the same issue. But I've just had a look, and I can't find a way to make them disappear, so that might not be the issue.

Probably an Electron/React issue with 10.9 - but I don't know enough about either of them to suggest much else, sorry.

**Comment by dpo** [comment]
I tried building from source, but the result is the same.

**Comment by dpo** [comment]
On OSX 10.10, the traffic lights are present.

**Comment by dpo** [comment]
Just in case it matters to anyone, the placeholder for the traffic lights is visible but seems empty on 10.9. The following screenshot is taken with two tabs open:
![screen shot 2016-10-20 at 13 29 55](https://cloud.githubusercontent.com/assets/38760/19574735/99c13bac-96d9-11e6-9a75-e3ae26f0b0ed.png)

**Comment by jankuca** [comment]
I am experiencing the same issue on OS X 10.9.5 (Mavericks).

Traffic lights are not visible:

![screen shot 2016-12-17 at 8 21 58 pm](https://cloud.githubusercontent.com/assets/367262/21289178/7de44650-c496-11e6-8e0e-45015b25e705.png)

After performing a "full reload", the controls are visible while the window background is, however, transparent:

![screen shot 2016-12-17 at 8 15 47 pm](https://cloud.githubusercontent.com/assets/367262/21289180/9a9e5e8e-c496-11e6-9b66-60c3ba106250.png)
